README.md

```diff
@@ -59,3 +59,4 @@ This value should be `true` if AWS S3 is used as storage.
 | AUTHORITY_CONTROL_BATCH_JOB_CHUNK_SIZE            | 100                           | Specify chunk size for authority control export job which will be used to query data from entities-links, and write to file                                                                           |
 | AUTHORITY_CONTROL_BATCH_ENTITIES_LINKS_CHUNK_SIZE | 100                           | Amount to retrieve per request to mod-entities-links                                                                                                                                                  |
 | MAX_UPLOADED_FILE_SIZE                            | 40MB                          | Specifies multipart upload file size                                                                                                                                                                  |
+| EUREKA_PERMISSIONS_MODEL                          | false                         | Specifies if eureka permissions model is used                                                                                                                                                         |
```

Comment:
It is more informative to use env variable PLATFORM=okapi|eureka with default value okapi.

Reply:
Updated